### PR TITLE
Clarify that no Span activation happens by default.

### LINKED ISSUE #238: Specify that the Start Span operation does not set active state
Currently, different implementations decide, on `Span` creation, whether to set it as the active/current instance or not (in Java `startSpan()` does not, but in Python `start_span()` does, for example).

I feel like there should be a clarification or recommendation for every language, mentioning the default **Start Span** operation does not set the active/current instance, and optionally offer an **Start Span + activate** operation*. This would give us some homogeneity.

If this makes sense, I'll open up a RFC. Opinions?

(In OT, we had two operations for this actually, namely **Start Span** and **Start Active Span**).

[1] In Java we cannot offer an **Start + Activate** method as it becomes hard to properly handle/report errors. See https://github.com/opentracing/opentracing-java/issues/291

### PR DESCRIPTION
**Pull request by carlosalberto**
Related to #238

### CODE REVIEW

File: specification/api-tracing.md [hunk 1]
```diff
@@ -223,8 +223,12 @@ directly. All `Span`s MUST be created via a `Tracer`.
 ### Span Creation
 
 Implementations MUST provide a way to create `Span`s via a `Tracer`, which is
-responsible for tracking the currently active `Span` and MAY provide default
-options for newly created `Span`s.
+responsible for tracking the currently active `Span` to be used as implicit
+parent, and MAY provide default options for newly created `Span`s.
```
Comment (tedsuo) [inline specification/api-tracing.md]: Small suggestion in language:
```markdown
Implementations MUST provide a way to create `Span`s via a `Tracer`. By 
default, the currently active `Span` is set as the new `Span`'s parent. The 
`Tracer` MAY provide other default options for newly created `Span`s.
```

File: specification/api-tracing.md [hunk 2]
```diff
@@ -223,8 +223,12 @@ directly. All `Span`s MUST be created via a `Tracer`.
...
+
+`Span` creation MUST NOT set the newly created `Span` as the currently
```
Comment (bogdandrutu) [inline specification/api-tracing.md]: s/currently active/current active?

File: specification/api-tracing.md (hunk 1 above)
Comment (carlosalberto) [reply specification/api-tracing.md]: I like it, thanks!

File: specification/api-tracing.md (hunk 2 above)
Comment (carlosalberto) [reply specification/api-tracing.md]: Actually both **currently active** and **current active** are used along this document ;)

File: specification/api-tracing.md (hunk 1 above)
Comment (carlosalberto) [reply specification/api-tracing.md]: Applied ;)

**Comment by Oberon00** [comment]
Reading the discussion at #238, maybe we should add a clarifying definition of the concepts of "started" vs "active" here?

**Comment by carlosalberto** [comment]
> maybe we should add a clarifying definition of the concepts of "started" vs "active" here?

I think that wouldn't hurt us, yes. I'd also leave that for another PR/Issue, in the spirit of making progress and have this PR merged ;)

**Comment by SergeyKanzhelev** [comment]
We have enough sign offs to merge this PR. Keeping the issue #238 active

**Pull request merged**